python/ray/tune/logger.py

```diff
@@ -84,9 +85,12 @@ def __init__(self,
                  config,
                  logdir,
                  upload_uri=None,
+                 use_default_loggers=True,
```

Comment:
Would it make more sense to change the argument to `loggers` instead of `use_default_loggers` and then just have a default be `loggers=(_JsonLogger, _TFLogger, _CSVLogger)`?

Reply:
That way could also get rid of the `custom_loggers` and just have a single loggers argument.

Reply:
oh I see; yeah let me make this change.